packages/in_app_purchase/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -26,8 +47,10 @@ const String _kOnBillingServiceDisconnected =
 /// some minor changes to account for language differences. Callbacks have been
 /// converted to futures where appropriate.
 class BillingClient {
-  BillingClient() {
+  BillingClient(PurchasesUpdatedListener onPurchasesUpdated) {
+    assert(onPurchasesUpdated != null);
```

Comment:
maybe assert _callback != null as well?

Reply:
Discussed offline, since this is set as non-null at construction time and never reassigned an assert isn't needed here.